sdks/python/apache_beam/metrics/cells.py

```diff
@@ -167,7 +169,7 @@ class DistributionCell(Distribution, MetricCell):
   """
   def __init__(self, *args):
     super(DistributionCell, self).__init__(*args)
-    self.data = DistributionData(0, 0, None, None)
+    self.data = DistributionAggregator.zero()
```

Comment:
Didn't we agree to call this "identity" or "empty"?

Reply:
Hmm yes, I made that change for Java. I'll make it for Python too. I'd like to do another PR for that.